index.html

```diff
@@ -2286,25 +2302,30 @@
           attribute MUST return the result of running the following steps:
         </p>
         <ol>
-          <li>If there is already an unsettled <a>Promise</a> <var>P</var> from
-          a previous call to get the <a for=
-          "PresentationReceiver">connectionList</a> attribute for the same <a>
-            PresentationReceiver</a> object, return <var>P</var> and abort all
-            remaining steps.
+          <li>If the <a>presentation controllers promise</a> is not
+          <code>null</code>, return the <a>presentation controllers promise</a>
+          and abort all remaining steps.
           </li>
-          <li>Let <var>P</var> be a new <a>Promise</a>.
+          <li>Otherwise, let the <a>presentation controllers promise</a> be a
+          new <a>Promise</a>.
           </li>
-          <li>Return <var>P</var>, but continue running these steps <a>in
-          parallel</a>.
+          <li>Return the <a>presentation controllers promise</a>, but continue
+          running these steps <a>in parallel</a>.
           </li>
-          <li>Let <var>list</var> be a new <a>PresentationConnectionList</a>.
-          </li>
-          <li>Resolve <var>P</var> with <var>list</var>.
+          <li>If the <a>set of presentation controllers</a> contains at least
+          one <a>presentation connection</a>, then run the following steps:
+            <ol>
+              <li>Let the <a>presentation controllers monitor</a> be a new <a>
```

Comment:
We might want to be explicit that the connections property of the new PresentationConnectionList should be populated with the contents of the presentation controllers monitor.